src/components/atoms/ChatMessage/ChatMessage.tsx

```diff
@@ -33,8 +34,14 @@ export const ChatMessage: React.FC<ChatProps> = ({
     <div className={containerStyles}>
       <div className="chat-message__text">{text}</div>
       <div className="chat-message__info">
-        <UserAvatar onClick={onAuthorClick} avatarSrc={author.pictureUrl} />
-        <span onClick={onAuthorClick} className="chat-message__author">
+        <UserAvatar
+          onClick={() => openUserProfileModal(author)}
+          avatarSrc={author.pictureUrl}
+        />
+        <span
+          onClick={() => openUserProfileModal(author)}
+          className="chat-message__author"
+        >
           {author.partyName}
         </span>
```

Comment:
Instead of having the same `onClick` handler on both `UserAvatar` and the `span`, it might be better to move it up to the parent `div`, and just have it in a single location?

We should also `useCallback` the handler to prevent re-renders from the inline function call changing on every render.